lib/supabase/server.ts

```
@@ -0,0 +1,23 @@
+import { cookies } from 'next/headers';
+import { createServerClient } from '@supabase/ssr';
+
+export function createClient() {
+  const cookieStore = cookies();
+
+  return createServerClient(process.env.NEXT_PUBLIC_SUPABASE_URL!, process.env.NEXT_PUBLIC_SUPABASE_ANON_KEY!, {
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)